Description: Disable byte-compile of python files
 This is already done by the debian packaging process
 .
 lofar (2.5.4-1trusty) trusty; urgency=low
 .
   * new upstream release
Author: Gijs Molenaar (launchpad ppa build key) <gijs@pythonic.nl>

--- lofar-2.5.4.orig/CMake/PythonInstall.cmake
+++ lofar-2.5.4/CMake/PythonInstall.cmake
@@ -81,16 +81,6 @@ macro(python_install)
     execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
       ${_py_abs} ${_build_dir}/${_py})
     install(FILES ${_py} DESTINATION ${_inst_dir}/${_py_path})
-    set(_py_code
-      "import py_compile"
-      "print('-- Byte-compiling: ${_inst_dir}/${_py}')"
-      "py_compile.compile('${_inst_dir}/${_py}', doraise=True)")
-    install(CODE 
-      "execute_process(COMMAND ${PYTHON_EXECUTABLE} -c \"${_py_code}\"
-                       RESULT_VARIABLE _result)
-       if(NOT _result EQUAL 0)
-         message(FATAL_ERROR \"Byte-compilation FAILED: ${_inst_dir}/${_py}\")
-       endif(NOT _result EQUAL 0)")
   endforeach(_py ${_py_files})
 
   # Make sure that there's a __init__.py file in each build/install directory.
